iwyu_tool.py

```diff
@@ -86,10 +86,21 @@ def clang_formatter(output):
     'clang': clang_formatter
 }
 
+
 def get_output(cwd, command):
     """ Run the given command and return its output as a string. """
+    # https://stackoverflow.com/a/4453495
+    # do not alter the current environment,
+    # but copy all of it for the child process,
+    # then prepend the directory of iwyu_tool.py to PATH
+    env = os.environ.copy()
```

Comment:
I'm not a huge fan of the Python "ask for forgiveness" adage, but I can't come up with a cleaner path append in short order.

Reply:
@cherti just suggested an alternative to me (see updated commit)